sessionctx/variable/varsutil.go

```diff
@@ -450,6 +450,14 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 			return value, errors.Errorf("tidb_max_chunk_size(%d) cannot be smaller than %d", v, maxChunkSizeLowerBound)
 		}
 		return value, nil
+	case TiDBOptJoinReorderThreshold:
+		v, err := strconv.Atoi(value)
+		if err != nil {
+			return value, ErrWrongTypeForVar.GenWithStackByArgs(name)
+		}
+		if v < 0 || v > 64 {
+			return value, errors.Errorf("tidb_join_order_algo_threshold cannot be smaller than 0 and larger than 63")
```

Comment:
`tidb_join_order_algo_threshold` -> `tidb_opt_join_reorder_threshold`, `and` -> `or`?

Reply:
how about:
```suggestion
			return value, errors.Errorf("%s cannot be smaller than 0 and larger than 64", TiDBOptJoinReorderThreshold)
```